src/swap/types.ts

```diff
@@ -27,6 +27,9 @@ export type BuildSwapTransactionParams = GetSwapQuoteParams & {
   fromAddress: Address; // The address of the user
 };
 
+/**
+ * Note: exported as public Type
+ */
 export type Fee = {
   amount: string; // The amount of the fee
   baseAsset: Token; // The base asset for the fee
```

Comment:
@0xAlec, let's change this to `token` instead of `baseAsset`, as in OnchainKit we call `asset`, `token`.